torchbenchmark/operator_loader/__init__.py

```diff
@@ -0,0 +1,145 @@
+import argparse
+import sys
+import types
+from typing import Any, Generator, List, Optional
+
+import torch
+from torch._dynamo.backends.cudagraphs import cudagraphs_inner
+from torch._inductor.compile_fx import compile_fx
+from torch._inductor.utils import gen_gm_and_inputs
+from torch._ops import OpOverload
+from torch.utils._pytree import tree_map_only
+
+from torchbenchmark.util.triton_op import (
+    BenchmarkOperator,
+    register_benchmark_mannually,
+)
+
+from .operator_inp_utils import aten, OperatorInputsLoader, to_channels_last
+
+timm_loader = OperatorInputsLoader.get_timm_loader()
```

Comment:
It is a bad idea to load operators during import. We should only load operators on-demand.

Reply:
fixed in https://github.com/pytorch/benchmark/pull/2495/commits/766027f73b37cc12a290c3bb04e6d716e438175d